cdap-standalone/pom.xml

```diff
@@ -265,6 +277,23 @@
                   <includeScope>runtime</includeScope>
                 </configuration>
               </execution>
+              <execution>
+                <id>copy-kafka-server-artifact</id>
+                <phase>prepare-package</phase>
+                <goals>
+                  <goal>copy-dependencies</goal>
```

Comment:
Instead of copy-dependencies, just use resource copy (again see how it copies cdap-spark-core).